packages/liveblocks-react/src/index.tsx

```diff
@@ -15,6 +15,7 @@ import {
 } from "@liveblocks/client";
 import * as React from "react";
 import useRerender from "./useRerender";
+export type { Json, JsonObject } from "@liveblocks/client";
```

Comment:
Why are we re exporting these types?

Reply:
I added these for convenience reasons, mostly. I found that in some projects, like our e2e tests, you already had an `import { … } from '@liveblocks/react'`, it was a bit annoying to have to add a new import line to import `Json` from `'@liveblocks/client'` instead of simply adding `Json` to the existing list of imports. Any objection to doing this?